config/features/grad_person_extended/config_split.patch.core.entity_form_display.node.person.default.yml

```diff
@@ -0,0 +1,177 @@
+adding:
+  dependencies:
+    config:
+      - field.field.node.person.field_grad_program_all_degree
+      - field.field.node.person.field_person_department
+      - field.field.node.person.field_person_university_id
+      - field.field.node.person.field_pt_student_advisor
+      - field.field.node.person.field_pt_student_cur_pos_inst
+      - field.field.node.person.field_pt_student_cur_pos_title
+      - field.field.node.person.field_pt_student_defense_date
+      - field.field.node.person.field_pt_student_program_start
+      - field.field.node.person.pt_student_dissertation_title
+    module:
+      - datetime
+  content:
+    created:
+      weight: 20
+    field_grad_program_all_degree:
+      type: options_select
+      weight: 15
+      region: content
+      settings: {  }
+      third_party_settings: {  }
+    field_meta_tags:
+      weight: 31
+    field_person_bio:
+      weight: 12
+    field_person_contact_information:
+      weight: 30
+    field_person_department:
+      type: entity_reference_autocomplete
+      weight: 14
+      region: content
+      settings:
+        match_operator: CONTAINS
+        match_limit: 10
+        size: 60
+        placeholder: ''
+      third_party_settings: {  }
+    field_person_education:
+      weight: 13
+    field_person_email:
+      weight: 8
+    field_person_hide:
+      weight: 29
+    field_person_hometown:
+      type: string_textfield
+      weight: 11
+      region: content
+      settings:
+        size: 60
+        placeholder: ''
+      third_party_settings: {  }
+    field_person_phone:
+      weight: 9
+    field_person_research_areas:
+      weight: 17
+      settings:
+        match_operator: STARTS_WITH
```

Comment:
It's in the original `grad_person_extended` split and not introduced by config split 2 work, but wondering if this was intended to change from `CONTAINS` to `STARTS_WITH`? If not, is it easier to change it here, or separately and re-export @pyrello?

Reply:
I'm guessing it got changed at some point between when this field started out in this split and then moved into the default config. I'd opt for as few differences as possible. Not sure about the best way to go about changing it off the top of my head. Possibly change it in the config file, run `drush @default.local cim`, then re-export? I'm assuming it would exclude it then.